pkgs/shells/any-nix-shell/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "any-nix-shell-${version}";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "haslersn";
+    repo = "any-nix-shell";
+    rev = "v${version}";
+    sha256 = "1fb4c2xkvk3hwy6km02h7j6wsmpachi66n286p2grzgk5k88pijr";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -r bin $out
+    wrapProgram $out/bin/any-nix-shell --prefix PATH ":" $out/bin
+    wrapProgram $out/bin/any-nix-shell-wrapper
+    wrapProgram $out/bin/nix-shell-info
```

Comment:
Why wrap the last two if you don't specify any action?

Reply:
They are bash scripts and wrapping them causes them to find bash.

Reply:
I think this happens automatically during ```fixupPhase``` (https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/setup-hooks/patch-shebangs.sh) and it has nothing to do with the ```makeWrapper```.

Reply:
You are exactly right and I removed the unnecessary wrappers.